src/components/sbb-title/readme.md

```diff
@@ -5,13 +5,13 @@
 
```

Comment:
Could you add a simple documentation about the purpose and usage of this component?